catalog/catalog.protocol.md

```diff
@@ -129,7 +129,7 @@ A [Dataset](../model/terminology.md#dataset) may contain 0..N [DCAT Distribution
 the distribution is obtained. Specifically, a `DataService` specifies the endpoint for initiating a [Contract Negotiation](../model/terminology.md#contract-negotiation) and [Transfer Process](../model/terminology.md#transfer-process).
 
 A Distribution may have 0..N `hasPolicy` attributes that contain an [ODRL `Offer`](https://www.w3.org/TR/odrl-vocab/#term-Offer) defining the [Usage Policy](../model/terminology.md#policy) associated with the [Dataset](../model/terminology.md#dataset) and this explicit `Distribution`.
-[Offers](../model/terminology.md#offer) must NOT contain any target attributes. The target of an [Offer](../model/terminology.md#offer) is the [Dataset](../model/terminology.md#dataset) that contains the distribution.
+**Offers must contain one root `target` property that is the id of the associated [Dataset](./terminology.md#dataset). Policy constraints must not contain `target` properties.** 
```

Comment:
as agreed in the meeting, change back

Reply:
@sebbader and I will discuss this the next days.
We also need to take care of schemas and how to deal with this. Schema should be separate PR.